submissions/Nemooochka/tiny-js-upd/index.js

```diff
@@ -0,0 +1,126 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+    constructor(species, name, gender, legs, saying, friends = []) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.legs = legs;
+        this.saying = saying;
+        this.friends = friends;
+        this.addFriends(friends);
+    }
+
+    addFriends(friends) {
+        if(Array.isArray(friends)) this.friends = [...this.friends, ...friends];
+        else this.friends.push(friends);
+    }
+
+    getFriendsName() {
```

Comment:
This method returns 1+ names. So the plural form (`Names`) in the method's name will better reflect the method implementation.